CHANGELOG.md

```diff
@@ -56,6 +56,11 @@ as necessary. Empty sections will not end in the release notes.
   Furthermore, Sentry integration can also be enabled and configured. And finally, it is now
   possible to configure the log level for specific loggers, not just the root logger. The old
   `logLevel` field is still supported, but will be removed in a future release.
+- If you are not using k8s and you are running Nessie with multiple nodes, you must either
+  * configure `nessie.version.store.persist.cache-invalidations.service-names`, see
+    [docs reference](https://projectnessie.org/nessie-latest/configuration/#version-store-advanced-settings),
+    or
+  * disable the reference cache by setting `nessie.version.store.persist.reference-cache-ttl` to `PT0S`.
```

Comment:
What's the benefit of enabling reference cache by default if users have to do additional config changes to run Nessie now?

Reply:
That's only if you run _multiple_ nodes _without_ k8s

Reply:
But is reference cache critical for single-node deployments?

"Must" is a strong word. It looks like the new default is going to cause undue failures for naive users when they scale Nessie.